docs/podman.1.md

```diff
@@ -167,8 +172,9 @@ the exit codes follow the `chroot` standard, see below:
 | [podman-top(1)](podman-top.1.md)          | Display the running processes of a container.                                  |
 | [podman-umount(1)](podman-umount.1.md)    | Unmount a working container's root filesystem.                                 |
 | [podman-unpause(1)](podman-unpause.1.md)  | Unpause one or more containers.                                                |
-| [podman-version(1)](podman-version.1.md)  | Display the Podman version information.                                        |
-| [podman-volume(1)](podman-volume.1.md)    | Manage Volumes.                                                                 |
+| [podman-varlink(1)](podman-varlink.1.md)  | Display the Podman version information.                                        |
+| [podman-version(1)](podman-version.1.md)  | Runs the varlink backend interface.                                            |
+| [podman-volume(1)](podman-volume.1.md)    | Manage Volumes.                                                                |
 | [podman-wait(1)](podman-wait.1.md)        | Wait on one or more containers to stop and print their exit codes.             |
```

Comment:
While you're in this file, can you remove the short '-s' option from '--storage-driver' as we no longer support that apparently.

Reply:
Fixed